src/mixed-flake.nix

```diff
@@ -17,16 +17,14 @@
 
               mkChildren = children: { inherit children; };
 
-              mkLeaf = leaf: { inherit leaf; };
-
             in
 
             rec {
 
               allSchemas = (flake.outputs.schemas or defaultSchemas) // schemaOverrides;
 
               # FIXME: make this configurable
-              defaultSchemas = (builtins.getFlake "https://api.flakehub.com/f/pinned/DeterminateSystems/flake-schemas/0.0.6+rev-8b320d78bfbc47515f3366b43e1a29c69519f36a/0189d657-3901-716b-9cde-7ec5aacc7cf8/source.tar.gz?narHash=sha256-3H1AXsE4SRqT1tZDb2gxdSwf3JInOgS1F8nq2QsXmHg=").schemas;
+              defaultSchemas = (builtins.getFlake "github:DeterminateSystems/flake-schemas/25fcca8e8baa402de593a3854fcc50581f4e912a").schemas;
```

Comment:
We'll need to swap to a flakehub URL before merging.

Reply:
We can't do that right now because we can't publish an updated `flake-schemas` : https://github.com/DeterminateSystems/flake-schemas/actions/runs/6023875150/job/16341456708

So we should merge this, publish a new version, and then update the URL.